src/main/resources/static/css/box.css

```diff
@@ -8,4 +8,94 @@
     display: flex;
     justify-content: center;
     margin-top: 90px;
-}
\ No newline at end of file
+}
+
+.custom-modal-margin {
+    margin-top: 150px;
+    margin-left: 28%;
+}
+
+.custom-modal-size {
+    max-width: 50%;
+    margin-top: 50px;
+}
+
+.years {
+    margin-left: 10px;
+}
+
+.custom-select-size {
+    height: 3rem;
+    width: 200px;
+}
+
+#selectedSubjectsContainer {
+    margin-top: 20px;
+    margin-left: 18px;
+}
+
+#subjects-container {
+    padding: 50px;
+}
+
+#subjects-container2 {
+    margin-top: 30px;
+}
+
+.modal-backdrop {
+    z-index: 1040 !important;
```

Comment:
if you need a huuuge z index and have to make it important then there is usually something wrong. where else do you habe a z-index set?

Reply:
I added when I created modals for error messages and it had problems working with subject form modal so that time I used z index. But now it seems to work only with opacity so I remove it